micro_dl/utils/meta_utils.py

```diff
@@ -53,15 +53,16 @@ def generate_normalization_metadata(
     # get arguments for multiprocessed grid sampling
     mp_grid_sampler_args = []
     for _, position in position_map:
-        mp_grid_sampler_args.append([position, True, grid_spacing])
+        mp_grid_sampler_args.append([position, grid_spacing])
 
     # sample values and use them to get normalization statistics
-    for i, channel in enumerate(channel_ids):
+    for i,channel in enumerate(channel_ids):
         show_progress_bar(
             dataloader=channel_ids,
             current=i,
             process="sampling channel values",
         )
+        print(channel)
```

Comment:
Is this intended?

Reply:
No. Accidentally left during testing. Removed. Thanks!